src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/V2/VacancyStatus.cshtml

```diff
@@ -1,24 +1,58 @@
-﻿@using SFA.DAS.EmployerAccounts.Web.Extensions
+﻿@using SFA.DAS.EAS.Portal.Client.Types;
 @model SFA.DAS.EmployerAccounts.Web.ViewModels.AccountDashboardViewModel
-
 @{
-    var vacancy = Model.AccountViewModel.SingleVacancy;
-}
+    Vacancy vacancy = null;
+
+    var actionLinkText = "";
+    int? numberOfApplications = null;
+    string status = "";
+
+    if (Model.HasSingleClosedVacancy)
+    {
+        vacancy = Model.AccountViewModel.Vacancies.First(v => v.Status == VacancyStatus.Closed);
+        actionLinkText = "Manage vacancy";
+        numberOfApplications = vacancy.NumberOfApplications;
+        status = "Closed";
+    };
 
+    if (Model.HasSinglePendingReviewVacancy)
+    {
+        vacancy = Model.AccountViewModel.Vacancies.First(v => v.Status == VacancyStatus.Submitted);
+        actionLinkText = "Preview vacancy";
+        status = "Pending review";
+    };
+
+    if (Model.HasSingleDraftVacancy)
+    {
+        vacancy = Model.AccountViewModel.Vacancies.First(v => v.Status == VacancyStatus.Draft);
+        actionLinkText = "Edit and submit vacancy";
+        status = "Draft";
+    };
+
+    if (Model.HasSingleRejectedVacancy)
```

Comment:
this looks like it includes work for CON-665 Rejected Vacancy

Reply:
Yes that's correct